src/client/Health/NodeView.jsx

```diff
@@ -1,22 +1,36 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
-import { Row, Container, Button, UncontrolledTooltip } from 'reactstrap'
-import { Link } from 'react-router-dom'
+import { FaEllipsisV } from 'react-icons/fa'
+import { Row, Container, Button, UncontrolledTooltip, UncontrolledPopover, PopoverHeader, PopoverBody } from 'reactstrap'
+import { useHistory, Link } from 'react-router-dom'
 import ReactFlow from 'react-flow-renderer'
 import { ConfigContext } from '../context/configContext.jsx'
 import PointsButton from '../South/PointsButton.jsx'
 import Modal from '../components/Modal.jsx'
+import ApiSchemas from '../North/Apis.jsx'
+import ProtocolSchemas from '../South/Protocols.jsx'
+import utils from '../helpers/utils'
+import NewApplicationRow from '../North/NewApplicationRow.jsx'
+import NewDataSourceRow from '../South/NewDataSourceRow.jsx'
+import { AlertContext } from '../context/AlertContext.jsx'
+import EditableIdField from '../components/EditableIdField.jsx'
+import validationNorth from '../North/Form/North.validation'
+import validationSouth from '../South/Form/South.validation'
 
 const colors = {
   border: {
     enabled: '1px solid #2ea948',
     disabled: '1px solid #eaecef',
     warning: '1px solid #ffc107',
+    warningg: '#eaecef5c',
```

Comment:
warningg or enabledd are not good variable names.

Reply:
in effect, it was just for some local testing , next i will removed it before push